plugins/opentelemetry/wiring.go

```diff
@@ -1,25 +1,68 @@
+//  1. a plugin to wrap the service with an OpenTelemetry wrapper to generate OT compatible traces by starting and stopping client spans for remote calls between services and correctly propagating context between services. The traces are then exported to a collector client such as jaeger or zipkin.
+//  2. a plugin to install an opentelemetry trace-based event logger for a go process. The logger adds all the logs as events to the current active span. If no active span exists, then no events are logged.
 //
-// The applications must use a backend.Tracer (runtime/core/backend) as the interface in the workflow.
+// Once the application is instrumented with these plugins, traces will be generated and collected by the selected collector. Traces will consist of spans generated by Plugin 1 with each span consisting of log events generated by Plugin 2.
+//
+// # Artifacts Generated
+//
+//  1. The package generates client and server side wrappers for instrumented services that contain opentelemetry instrumentation (context propagation, creation of spans). The generated clients handle context propagation correctly on both the server and client sides.
+//
+// # Wiring Usage
+//
+// To instrument services with opentelemetry instrumentation:
+//
+//	opentelemetry.Instrument(spec, "my_service", "collector_name")
+//
+// Calling [Instrument] will generate client and server side wrappers responsible for starting and stopping spans as well as correctly propagating the context across service boundaries.
+//
+// To redirect a process's logging statements to opentelemetry generated spans:
+//
+//	opentelemetry.Logger(spec, "my_process")
+//
+// Calling [Logger] will redirect all logging statements generated by the process to opentelemetry where the log statements will be converted into opentelemetry Events and added to the list of events for the current active span.
+//
+// In order to generate complete end-to-end traces of the application, all services of the application need to be instrumented with OpenTelemetry.
+// If the plugin is only applied to a subset of services, the application will run, but the traces produced won't be end-to-end and won't be useful.
+// The plugin does not support instrumenting clients for backends such as databases, caches, queues, etc. If needed, please consider submitting a PR or contacting the maintainers via google groups or on slack.
+//
+// Example usage (for complete instrumentation):
+//
+// # Wiring Example
+//
+//	func applyOTelOptions() {
+//		jaeger.Container(spec, "jaeger")
+//		for _, service := range serviceNames {
+//			opentelemetry.Instrument(spec, service)
+//		}
+//		for _, proc := range processNames {
+//		opentelemetry.Logger(spec, proc)
+//		}
+//	}
+//
+// See the [ot_logger] wiring spec for the Leaf application for a complete example
+//
+// # Accessing Traces
+//
+// The traces are generated and sent to the zipkin or jaeger collector.
```

Comment:
Explain, at least, that the collector will expose a web UI for inspecting traces, and it depends on the collector used.  You could even just say "for zipkin, port XXX, for jaeger, port XXX".  At the least, provide links to the plugins.